src/picknik_ur_base_config/config/moveit/ur_servo.yaml

```diff
@@ -32,7 +29,6 @@ publish_joint_accelerations: false
 ## Plugins for smoothing outgoing commands
 use_smoothing: true
 smoothing_filter_plugin_name: "online_signal_smoothing::ButterworthFilterPlugin"
-low_pass_filter_coeff: 1.5  # Larger --> trust the filtered data more, trust the measurements less.
```

Comment:
while it is gone from here, it can still be passed in as follows: https://github.com/ros-planning/moveit2/pull/2114/files

... unsure how we should handle this going forward, though.